kaitai_utils.py

```diff
@@ -45,6 +46,28 @@ def _get_metadata():
         return yaml.safe_load(file)['meta']
 
 
+def object_has_process_key(searchable_object):
+    if type(searchable_object) is dict:
+        for key in searchable_object.keys():
+            if 'process' in searchable_object:
+                return True
+            if type(searchable_object[key]) in [list, dict]:
+                if object_has_process_key(searchable_object[key]):
+                    return True
+    elif type(searchable_object) is list:
+        for item in searchable_object:
+            if type(item) is list or type(item) is dict:
```

Comment:
You could do the same here as on line 54:
```suggestion
            if type(item) in [list, dict]:
```

Reply:
done!